src/main/kotlin/no/nav/familie/ef/sak/repository/domain/Søknad.kt

```diff
@@ -0,0 +1,83 @@
+package no.nav.familie.ef.sak.repository.domain
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.kontrakter.ef.søknad.SøknadBarnetilsyn
+import no.nav.familie.kontrakter.ef.søknad.SøknadOvergangsstønad
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Embedded
+import org.springframework.data.relational.core.mapping.MappedCollection
+import org.springframework.data.relational.core.mapping.Table
+import java.util.*
+
+@Table("gr_soknad")
+data class Søknad(@Id
+                  val id: UUID = UUID.randomUUID(),
+                  @Column("behandling_id")
+                  val behandlingId: UUID,
+                  val type: SøknadType,
+                  @Column("soknad")
+                  val søknad: ByteArray,
+                  @Column("saksnummer_infotrygd")
```

Comment:
Undestrek er default omskrivning i spring data for camel case, så @Column er egentlig ikke nødvendig.

Reply:
909118b